addons/mail_group/tests/test_mail_group.py

```diff
@@ -78,6 +78,51 @@ def test_find_member(self):
         member = self.test_group._find_member(email, partner_2.id)
         self.assertFalse(member, 'Should not return any member because the only one with the same email has a different partner')
 
+    def test_find_member_for_alias(self):
+        """Test the matching of a mail_group.members, when 2 users have the same partner email, and
+        that the first user was subscribed."""
+        User = self.env['res.users'].with_context(no_reset_password=True)
+        email = 'Alias <test_find_member_for_alias@example.com>'
+
+        alias = self.env['mail.alias'].create({
+            'alias_name': 'test_find_member_for_alias',
+            'alias_model_id': self.env.ref('base.model_res_partner').id,
+            'alias_contact': 'followers',
+        })
+
+        user = User.create({
```

Comment:
Well you should use existing data from mail common (user_employee, user_admin, ...). That way we avoid doing a lot of boiler plate just for some small tests.

Reply:
because the order in which the user is created is important, and so it more explicit to see here that we have 2 users with same email, and that we subscribe with the first one.

Reply:
Ah ok, did not understand it, this could be specified in test docstring :) . Then you may use the ``_mail_new_test_user`` tool, that sets up the relevant context keys to speedup creation of users (+ adding the groups, ... ) .